front/src/utils/date.ts

```diff
@@ -242,3 +242,22 @@ export const formatDateString = (date?: Date | null) => {
   if (!date) return '';
   return dayjs(date).format('DD/MM/YY');
 };
+
+const parseDate = (dateString: string): Date => {
+  const [datePart, timePart] = dateString.split(' ');
+  const [day, month, year] = datePart.split('/').map((part) => part.padStart(2, '0'));
+  const isoDateString = `${year}-${month}-${day}T${timePart || '00:00:00'}`;
+  return new Date(isoDateString);
+};
+
+export const formatDepartureDate = (date: string): string => {
+  const locale = i18n.language;
+  const options: Intl.DateTimeFormatOptions = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  };
+
+  return new Intl.DateTimeFormat(locale, options).format(parseDate(date));
```

Comment:
Nit: no need to pass `locale` here, `undefined` will use the default.

Reply:
done, but don't we lost the display in English ?